server/src/session.ts

```diff
@@ -830,6 +834,12 @@ export class Session {
     } else {
       // Immediately enable Legacy/ViewEngine language service
       this.info(`Enabling VE language service for ${projectName}.`);
+      if (!this.ivy && this.angularCoreVersion.major >= 9) {
```

Comment:
We should resolve the version of `@angular/core` here, since we already know that `@angular/core/core.d.ts` exists. This would allow us to run the check for each project.